git_cheatsheet.md

```diff
@@ -7,7 +7,7 @@
 - add
   - `git add foo.py`: add `foo.py` to the staging area
   - `git add foo.py bar.py`: add `foo.py` AND `bar.py` to the staging area
-  - `git add -u`: Adds all tracked files to the staging area.
+  - `git add -u` or `git add .`: Adds all tracked files to the staging area.
```

Comment:
the problem with "git add ." is that it depends on your pwd. that is not the case with "git add -u"